src/components/AccountForm/index.tsx

```diff
@@ -0,0 +1,250 @@
+import { ChangeEvent, useRef, useState } from 'react';
+
+import { zodResolver } from '@hookform/resolvers/zod';
+import classNames from 'classnames/bind';
+import { FormProvider, useForm } from 'react-hook-form';
+import { z } from 'zod';
+
+import { ERROR_MESSAGE, INPUT_NAMES, REGEX } from '@/constants';
+
+import Avatar from '@/components/commons/Avatar';
+import { BaseButton } from '@/components/commons/buttons';
+import { InputField } from '@/components/commons/inputs';
+import { ConfirmModal, ModalButton } from '@/components/commons/modals';
+import { USER_DATA } from '@/constants/mockData/headerMockData';
+import useMultiState from '@/hooks/useMultiState';
+
+import styles from './AccountForm.module.scss';
+
+const cx = classNames.bind(styles);
+
+const { profileImageUrl, nickname, password, passwordConfirm, image } = INPUT_NAMES;
+
+const ImageSchema = z.object({
+  [image]: z.instanceof(File),
+});
+
+const ProfileSchema = z.object({
+  [profileImageUrl]: z.string().optional(),
+  [nickname]: z.string().min(1, { message: ERROR_MESSAGE.nickname.min }),
+});
+
+const PasswordSchema = z
+  .object({
+    [password]: z
+      .string()
+      .min(8, { message: ERROR_MESSAGE.password.min })
+      .regex(REGEX.password, ERROR_MESSAGE.password.regex),
+    [passwordConfirm]: z.string(),
+  })
+  .refine((data) => data.password === data.passwordConfirm, {
+    path: [passwordConfirm],
+    message: ERROR_MESSAGE.password.refine,
+  });
+
+const AccountForm = () => {
+  const { multiState, toggleClick } = useMultiState(['resetConfirmModal', 'saveAlertModal']);
+  const [newImageUrl, setNewImageUrl] = useState(USER_DATA.profileImageUrl || '');
+  const fileInputRef = useRef(null);
+  const buttonRef = useRef<HTMLButtonElement>(null);
+
+  const Profilemethods = useForm({
```

Comment:
```suggestion
  const profileMethod = useForm({
```

p랑 m이 대소문자가 바뀐것 같습니다!